### PR TITLE
Rubric title and description are now read out to screen reader

### PR DESCRIPTION
**Pull request by MitchelPaulin**
Rubric title and description are now both read rather than just the title, went with a lang term with a period to create a pause between the two

### CODE REVIEW

**Comment by MitchelPaulin** [comment]
See this thread for context on the `ci.yml` change https://learningoutcomes.slack.com/archives/C0ANRQHFU/p1652458024335869

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ on: pull_request
 jobs:
   lint:
     name: Lint
-    timeout-minutes: 5
+    timeout-minutes: 10
```
Comment (imcdonald-d2l) [inline .github/workflows/ci.yml:6]: Have we tried the increased timeout without the `npm i --verbose` update? 🤔

Comment (MitchelPaulin) [reply .github/workflows/ci.yml:6]: --verbose just gives more info, it doesn't effect npm execution

Comment (imcdonald-d2l) [reply .github/workflows/ci.yml:6]: I was curious if we could revert back to `npm ci` specifically for marginally improved CI time. Assuming the timeout was the only issue.

Comment (MitchelPaulin) [reply .github/workflows/ci.yml:6]: Oh, not sure, I can try

**Review by imcdonald-d2l — APPROVED** [review]

**Pull request merged**